python/ray/air/util/tensor_extensions/utils.py

```diff
@@ -28,16 +30,52 @@ def _is_ndarray_variable_shaped_tensor(arr: np.ndarray) -> bool:
     return True
 
 
-def _create_possibly_ragged_ndarray(
+@PublicAPI(stability="beta")
```

Comment:
curious shouldn't we start with `alpha`?

Reply:
Fixed